solutions/usaco-760.mdx

```diff
@@ -118,3 +119,36 @@ int main(){
 
 </CPPSection>
 </LanguageSection>
+
+<LanguageSection>
```

Comment:
why another `<LanguageSection>`?

Reply:
not addressed

Reply:
Arent the time complexities different though?